README.md

```diff
@@ -15,72 +15,75 @@ riff System contains three API groups with CustomResourceDefinitions that are th
 - `core.projectriff.io/v1alpha1`
   - `Deployer` - deployers map HTTP requests to applications, functions, containers or images with Kubernetes core resources
 - `streaming.projectriff.io/v1alpha1`
-  - `Provider` - an abstraction over a message broker
   - `Stream` - streams of messages
-  - `Processor` - processors apply functions to messages on streams
+  - `Processor` - processors apply functions, containers or images to messages on streams
+  - `KafkaProvider` - kafka based stream provider
 - `knative.projectriff.io/v1alpha1`
   - `Adapter` - adapters map applications, functions or container images into an existing Knative Service or Configuration.
   - `Deployer` - deployers map HTTP requests to applications, functions, containers or images with Knative
 
-### Runtime
+### Runtimes
 
-A `controller` and `webhook` Deployments exist in the `riff-system` namespace to validate and reconcile the riff CRDs.
+Each runtime has an independent manager deployment that reconciles and hosts validation and defaulting webhooks for the CRDs needed by that runtime.
+
+The build component is required by each runtime. 
```

Comment:
This kinda contradicts the the "independent" sentence above, although the word "runtime" (of which `build` isn't) hasn't been introduced in that README yet, AFAICT

Reply:
Maybe just state that core, streaming and knative are "runtimes" as the first sentence of that section.